src/java.desktop/share/classes/javax/swing/JMenu.java

```diff
@@ -1256,7 +1256,11 @@ public void applyComponentOrientation(ComponentOrientation o) {
     }
 
     /**
-     * {@inheritDoc}
+     * Sets the orientation for this menu and the associated popup component
```

Comment:
```suggestion
     * Sets the orientation for this menu and the associated popup menu
```
I think *popup component* isn't specific enough.

Maybe “and *its* associated…” is even better?